Formula/go@1.9.rb

```diff
@@ -0,0 +1,105 @@
+class GoAT19 < Formula
+  desc "Go programming environment (1.9)"
+  homepage "https://golang.org"
+
+  stable do
+    url "https://dl.google.com/go/go1.9.4.src.tar.gz"
+    mirror "https://fossies.org/linux/misc/go1.9.4.src.tar.gz"
+    sha256 "0573a8df33168977185aa44173305e5a0450f55213600e94541604b75d46dc06"
+
+    go_version = version.to_s.split(".")[0..1].join(".")
+    resource "gotools" do
+      url "https://go.googlesource.com/tools.git",
+          :branch => "release-branch.go#{go_version}"
+    end
+  end
+
+  bottle do
+    sha256 "f888bb85ad375f7e7ee683e1d575de4ec404a9a26f528bda36aa38a4d9ff5915" => :high_sierra
+  end
+
+  head do
```

Comment:
`head` block should be pinned to the 1.9 branch or removed entirely.

Reply:
Removed is good

Reply:
removed